src/runner/build.js

```diff
@@ -2,6 +2,8 @@ import webpack from 'webpack'
 import { logError, log } from '../util/log'
 
 export default (saguiOptions) => new Promise((resolve, reject) => {
+  process.env.NODE_ENV = 'production'
```

Comment:
Build always sets the environment to production. I feel this is a sensible default that we can change if a problem arises. 

I've done similar approaches for other actions.